### DISCUSSION TITLE
Sensor Request - Help: Sensirion SEN55/5x - Particle and VOC/NOX

### DISCUSSION THREAD

**Discussion by tyeth** [arendst/Tasmota#17646] [Ideas]
### PROBLEM DESCRIPTION
I wish to use the Sensirion SEN5x series of sensors, ideally the SEN55 which includes the NOx sensor in addition to VOC and PPM. (Sen54 = PPM+VOC, SEN50=PPM)
Product page contains links to info and code, as does sensirions github repo.
https://sensirion.com/products/catalog/SEN55/

I bought mine from a UK supplier for £20, without a JST-GHR 6-Pin cable, which meant a separate shipment but saved a fair bit of money as combined units were £50 elsewhere.
https://www.anglia-live.com/product/2080075001/SEN55-SDN-T/SENSIRION-AG

I'd like to attempt to add it, but need help understanding if it is acceptable to add a dependency on Sensirions driver?

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): T-DisplayS3 and maybe sonoffs in future
- [x] Tasmota binary firmware version number used: n/a
  - [x] Pre-compiled
  - [x] Self-compiled
- [x] Flashing tools used: web
- [x] Provide the output of command: `Backlog Template; Module; GPIO 255`: ping pong
```lua
  Configuration output here:
n/a

```
- [x] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:
n/a

```
- [x] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:
ping pong

```
- [x] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:
ping pong

```

### TO REPRODUCE
No steps required

### EXPECTED BEHAVIOUR
Magically works

### SCREENSHOTS
n/a

### ADDITIONAL CONTEXT
I'm volunteering with the proviso I need a little help.
I found the initial request as a discussion after fully processing the issue requirements:
https://github.com/arendst/Tasmota/discussions/15463

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Jason2866** [comment]
Please do not open an issue for a Feature Request or a duplicate discussion.
Discussed further in #15463
